src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/ItemContainerGenerator.cs

```diff
@@ -2061,7 +2061,8 @@ void AddEmptyGroupItem(CollectionViewGroup group)
 
             // add it to the list of placeholder items (this keeps it from being GC'd)
             if (_emptyGroupItems == null)
-                _emptyGroupItems = new ArrayList();
+                _emptyGroupItems = new();
```

Comment:
Could you specify the type here. I believe this repo follows the dotnet/runtime coding style (See [here](https://github.com/dotnet/wpf/blob/main/Documentation/acceptance_criteria.md#general-guidelines)) which says to only use `var` or `new()` when the type is apparent (Rule 10 [here](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md)).

Also could you check your other PRs for this, I think a saw a couple places with similar style.

Note: I'll open a PR to update the links that points to old version of dotnet/runtime documentation files.

Reply:
> @ThomasGoulet73 I believe this repo is still in 2004 unfortunately, haha.

I agree 😄

> Nonetheless, I totally agree with you, especially as I hate `var` and I prefer to use `new()` only when the type is declared on the same line as well. Will review some of my other PRs.

Thanks for looking into it.